region/maridia/outer/Mama Turtle Room.json

```diff
@@ -886,6 +902,46 @@
       ],
       "flashSuitChecked": true,
       "note": "To save energy, start from the upper right ledge; it may require repositioning Mama Turtle to the right to get to the ledge."
+    },
+    {
+      "link": [5, 6],
+      "name": "Base",
+      "requires": [
+        "canUseGrapple",
+        "SpaceJump"
+      ]
+    },
+    {
+      "link": [6, 3],
+      "name": "Base",
+      "requires": []
+    },
+    {
+      "link": [6, 5],
+      "name": "Base",
+      "requires": [
+        "canUseGrapple",
```

Comment:
The problem is if you miss the grapple, you fall and die. HiJump makes it a bit easier

Reply:
yup, updated